src/ListContainer.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import { useSelector, useDispatch } from 'react-redux';
+import ListContainer from './ListContainer';
+
+jest.mock('react-redux');
+
+beforeEach(() => {
+  const tasks = [
+    { id: 1, title: 'Task-1' },
+  ];
+
+  useSelector.mockImplementation((selector) => selector({
+    tasks,
+  }));
+
+  jest.clearAllMocks();
+});
+
+describe('ListContainer', () => {
+  const dispatch = jest.fn();
+
+  useDispatch.mockImplementation(() => dispatch);
+
+  it('renders tasks', () => {
+    const { getByText } = render((
+      <ListContainer />
+    ));
+
```

Comment:
```suggestion
    const { getByText } = render(<ListContainer />);
```

간단한 코드는 한줄로 해줘도 좋을 것 같아요.

줄바꿈을 할 때 자신만의 기준이나 논리가 있나요?

저 같은 경우 가독성을 고려해서 `props` 가 2 개 이상이거나 이름이 너무 긴 경우 줄바꿈을 해주는 편이에요.

빈 줄을 넣어줄 때도 의미상으로 구분될 때 넣어주는 편이에요.

빈 코드 한 줄에도 의도가 담겨있다는 걸 생각해주시면 좋을 것 같아요!